x-pack/packages/kbn-elastic-assistant/impl/assistant/api/prompts/bulk_update_prompts.ts

```diff
@@ -47,7 +47,13 @@ export const bulkUpdatePrompts = async (
     toasts?.addError(error.body && error.body.message ? new Error(error.body.message) : error, {
       title: i18n.translate('xpack.elasticAssistant.prompts.bulkActionspromptsError', {
         defaultMessage: 'Error updating prompts {error}',
-        values: { error },
+        values: {
+          error: error.message
```

Comment:
Found an issue when handling error, please find the item 4 in https://github.com/elastic/kibana/pull/187705#discussion_r1674452482